docs/general/administration/hardware-selection.md

```diff
@@ -88,89 +86,56 @@ When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will r
 
 ### System Memory (RAM)
 
-As a general rule of thumb, at least 8GB is recommended, though significantly smaller amounts may work.
+As a general rule of thumb, at least 8GB is recommended for most configurations. For a Linux server running without a GUI, 4GB should be enough.
 
-:::tip Integrated Graphics
-
-When using Integrated graphics, the integrated graphics processer usually uses a part of system memory as video memory. It is recommended that a dual channel configuration be used and additional capacity be installed.
-
-:::
+It is recommended to add more memory on Windows 11 due to the OS being heavier.
 
 ### Graphics Cards (GPUs)
 
 With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
 
-Encoder Quality: Intel > Nvidia > AMD > Apple
-
-Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.
+Encoder Quality: Apple ≥ Intel > Nvidia >>> AMD
```

Comment:
The same as above. This is not constantly true and is depending on the input videos.